Library/Homebrew/livecheck.rb

```diff
@@ -1,4 +1,4 @@
-# typed: true
+# typed: strict
```

Comment:
`typed: strict` is not desirable. It adds a lot of `T.let` boilerplate that harms readability. Instead we should consider enabling the sorbet-rubocop cop to require type signatures to all methods.